README.md

```diff
@@ -61,82 +66,64 @@ that can be used as a dependency in any other wallet based project. It is develo
-[Development Setup Guide](https://github.com/openMF/mobile-wallet/wiki/Development-Setup)
+Please refer to the  [Development Setup Guide](https://github.com/openMF/mobile-wallet/wiki/Development-Setup) for detailed instructions on configuring the development environment.
 
 ### **Committing Your Changes**
-
-Once you make changes in your local repository, you need to commit them to your GitHub repository.  
-Don't know how to commit? Click [here](https://github.com/openMF/mobile-wallet/wiki/Committing-Your-Changes) to learn more.
+After making changes in your local repository, you will need to commit them to your GitHub repository.  
+If you are unfamiliar with the process of committing changes, please refer to the [Committing Your Changes](https://github.com/openMF/mobile-wallet/wiki/Committing-Your-Changes) guide.
 
 ### **Making a Pull Request**
-
-After pushing your changes to your forked repository, you can create a pull request to propose merging your changes into the main project.  
-Don't know how to create a pull request? Click [here](https://github.com/openMF/mobile-wallet/wiki/Making-a-Pull-Request) to learn more.
+Once your changes have been pushed to your forked repository, you can create a pull request to propose integrating your updates into the main project.  
+For guidance on creating a pull request, please visit the [Making a Pull Request](https://github.com/openMF/mobile-wallet/wiki/Making-a-Pull-Request) guide.
 
 ### **Squashing Your Commits**
+To ensure a clean and organized Git history, contributors are encouraged to squash their commits before merging.  
+Instructions on how to squash commits can be found in the [Squashing Your Commits](https://github.com/openMF/mobile-wallet/wiki/Squashing-Your-Commits) guide.
 
-To maintain a clean and organized Git history, it is recommended to squash your commits before merging.  
-Don't know how to squash commits? Click [here](https://github.com/openMF/mobile-wallet/wiki/Squashing-Your-Commits) to learn more.
-
-### **Solving Merge Conflicts**
-
-Sometimes, your pull request may encounter merge conflicts that need to be resolved manually.  
-Don't know how to resolve merge conflicts? Click [here](https://github.com/openMF/mobile-wallet/wiki/Solving-Merge-Conflicts) to learn more.
+### **Resolving Merge Conflicts**
+Occasionally, merge conflicts may arise when your pull request is being reviewed. These conflicts need to be resolved manually.  
+To learn how to resolve merge conflicts, please refer to the [Solving Merge Conflicts](https://github.com/openMF/mobile-wallet/wiki/Solving-Merge-Conflicts) guide.
 
 ### Conclusion
 By following these contribution guidelines, you're all set to start contributing to the Mobile Wallet (Mifos Pay) project. We appreciate your efforts and look forward to your valuable contributions. Happy coding!
 
 ## Instructions to get the latest APK
-
 To get the latest apk fom the Github actions artifacts, follow these steps:
-
```

Comment:
https://github.com/openMF/mobile-wallet/actions/workflows/master_dev_ci.yml
use this link to download artifacts for all platform and mention steps to download application for different platform.